test/test_unistd.rs

```diff
@@ -104,6 +104,57 @@ mod linux_android {
     }
 }
 
+#[test]
+#[cfg(not(any(target_os = "ios", target_os = "macos")))]
```

Comment:
Because the `getgroups()` and `setgroups()` calls don't work as expected on Apple platforms, as discussed in the docs.